en/theme/material/base.html

```diff
@@ -0,0 +1,297 @@
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Instead overriding the full `base.html` shall we try to use the extension points?